docs/docs/guides/repository.mdx

```diff
@@ -108,7 +108,8 @@ Using the GraphQL Interface, it is possible to add a `Repository` or `Read-only
       data: {
         name: {value: "My Git Repository"},
         location: {value: "https://GIT_SERVER/YOUR_GIT_USERNAME/YOUR_REPOSITORY_NAME.git"},
-        # credential: {id: "CREDENTIAL_ID_FROM_PREVIOUS_REQUEST"} <-- optional: This needs to be the credential id created at step 2
+        # The HFID returned during step 2 will be the name of the credentials
```

Comment:
This line is a bit long so it causes the user to have to side-scroll in the docs to see all the text. It would be a bit cleaner to spread it out on two comment lines.

(It looks like we already have the same "problem" in the docs that describe how to create a repo with the SDK. Also on the SDK the credential is marked as optional for the CoreRepository too (i.e. not only the ReadOnlyRepository). Technically it is optional but I'd say that it's highly recommended if you create branches that are setup to sync with Git)

Reply:
Yep, fixed!